File: engine/engine-model/src/main/scala/tools/packet/wedding/WeddingPartnerTransfer.scala
package tools.packet.wedding

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class WeddingPartnerTransfer(private var _partnerId: Int, private var _mapId: Int) extends PacketInput {
  def partnerId: Int = _partnerId

  def mapId: Int = _mapId

  override def opcode(): SendOpcode = SendOpcode.NOTIFY_MARRIED_PARTNER_MAP_TRANSFER
}

File: engine/engine-model/src/main/java/tools/packet/wedding/WeddingPartnerTransfer.java
package tools.packet.wedding;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record WeddingPartnerTransfer(Integer partnerId, Integer mapId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.NOTIFY_MARRIED_PARTNER_MAP_TRANSFER;
   }
}